### DISCUSSION TITLE
I have never passed a plot filter. HELP!

### DISCUSSION THREAD

**Discussion by Derekbps2003** [Chia-Network/chia-blockchain#4379] [Q&A]
I have been farming for about a week and have spent countless hours looking at the screen while plotting. I have never seen any of my plots pass a single filter. Would that be because I've done something wrong? Or is it that I'm still very small storage wise? ( I haven't completed all of my plotting yet).

**Comment by SackofFluor** [comment]
Check the log files first und maybe switch the logging-mode to info in the configuration file
How many plots do you have?

**Reply by Derekbps2003** [reply]
How would I do that? I only have 20 plots so far. But I have about 50tb worth of drives left to fill still.

**Reply by YahorL** [reply]
I think you have not enough plots. Only 1/512 pass the filter. If your plots have the label "farming", everything should be okay.

**Reply by Derekbps2003** [reply]
Ok great thank you for the help!

**Reply by Keriwaza** [reply]
With 20 plots I saw plots pass filter every once in a while.  Saying he needs more plots to get one pass filter is not totally accurate - he may have an issue with his installation. You can pass filter with one plot, it is just less likely that you'll see it very often given the net space. 

I would suggest you change your logging level to INFO (from WARNING), and review the messages in the log to make sure your installation is healthy. You should also verify that you have port 8444 open to your farmer or your router recognizes your upnp to your farmer on that port (look at upnp on router). You should see the farmer's IP address there as well as the port 8444. Cheers

**Reply by Derekbps2003** [reply]
Yes my port 8444 is open. And I have changed logging level to info, Could you help me out? Since I do not understand code at all.

**Reply by matthewjbauer** [reply]
I have 17 plots and counting. I started Chia about 2 weeks ago and just started farming  (again) about an hour ago. I had an install on my farm server that just refused to sync up. I got the node synced and the wallet is syncing finally and is close to synced. I have seen so far about 3x of 1/17 passed filter in the last hour (that is how long I have been synced to the node and actively farming again). I am 100% certain that having more plots increases your odds of getting a good plot that will pass filter and win, but to those that think simply having more plots is the only key to getting a plot to 1/x or 2/x passed filter etc, that is absolutely false and I am witnessing that right now.
